dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessors.java

```diff
@@ -12,15 +12,34 @@
 public class DbParameterAccessors implements Iterable<DbParameterAccessor> {
```

Comment:
Could it be worth renaming this class to something like `DbAttributeAccessor` as it's not only used for stored procedure parameters?